Sources/Sentry/SentryClient.m

```diff
@@ -607,7 +608,7 @@ - (void)setSdk:(SentryEvent *)event
     id integrations = event.extra[@"__sentry_sdk_integrations"];
     if (!integrations) {
         integrations = [NSMutableArray new];
-        for (NSString *integration in self.options.enabledIntegrations) {
+        for (NSString *integration in SentrySDK.currentHub.installedIntegrationNames) {
```

Comment:
l: Im not sure about this cyclic reference. Can we do this without SentryClient knowing about SentryHub?

Reply:
Why is this a cyclic reference? It all works fine, there are no problems? 🤔

Reply:
Because SentryHub has a SentryClient and now SentryClient access SentryHub. I would try to avoid this kind of thing, but I know we have this all around the SDK. 
You can keep it this way.

Reply:
I guess we could store the installedIntegrationNames somewhere else, but I think the options isn't the right place either. If you're happy with the solution as it stands now, then I am happy too. Otherwise we find another place.

Reply:
Yeah, lets keep this way